.rubocop.yml

```diff
@@ -1,9 +1,18 @@
-require: 
+require:
   - rubocop-performance
-  - rubocop-rails
 
 inherit_from: .rubocop_todo.yml
 
+inherit_gem:
+  rubocop-govuk:
+    - config/default.yml
+    - config/rails.yml
+    - config/rspec.yml
+
+inherit_mode:
+  merge:
+    - Exclude
+
 AllCops:
   TargetRubyVersion: 2.7
```

Comment:
Should we have updated this to 3.x? 🤔

Reply:
Good spot, I completely missed that. I'll remove it, if it's not present then rubocop falls back on `.ruby-version`. There doesn't seem to be any benefit in configuring it in multiple places 🤷

Reply:
Me too - I've updated ruby multiple times and not spotted it 😬 
Thanks!